tests/test_utils.py

```diff
@@ -59,6 +59,41 @@ def test_api_url_for(self):
         with self.app.test_request_context():
             assert api_url_for('dummy_view', pid='123') == '/api/v1/123/'
 
+    def test_api_v2_url_with_port(self):
+        """Handles URLs with port correctly"""
```

Comment:
As per COSDev, docstrings for test methods are discouraged in favor of long test method names and code comments.

> Use long, descriptive names. This often obviates the need for doctrings in test methods. This also makes it easier to locate tests that fail.

https://cosdev.readthedocs.org/en/latest/process/testing.html#general-testing-guidelines